backend/scrapers/em2.js

```diff
@@ -0,0 +1,371 @@
+          res.venueEventUrl = eventEl.querySelector('.more-button')?.href ?? null;
+
+          // TODO: Hoe te bepalen of show is uitverkocht?
+          const uaRex = new RegExp(unavailabiltyTerms.join('|'), 'gi');
+          res.unavailable = !!eventEl.textContent.match(uaRex);
+          res.soldOut =
+            !!eventEl
+              .querySelector('.program__status')
+              ?.textContent.match(/uitverkocht|sold\s?out/i) ?? false;
+
+          res.longText = eventEl.querySelector('.agenda-item-content p')?.textContent ?? null; // tijdelijk om in te controleren
+          return res;
+        }),
+    {
+      workerData,
+      unavailabiltyTerms: terms.unavailability,
+    },
+  );
+
+  rawEvents = rawEvents.map(this.isMusicEventCorruptedMapper);
+
+  const eventGen = this.eventGenerator(rawEvents);
+  // eslint-disable-next-line no-unused-vars
+  const checkedEvents = await this.rawEventsAsyncCheck({
+    eventGen,
+    checkedEvents: [],
+  });  
+
+  this.saveBaseEventlist(workerData.family, checkedEvents);
+    
+  const thisWorkersEvents = checkedEvents.filter(
+    (eventEl, index) => index % workerData.workerCount === workerData.index,
+  );
+    
+  return this.mainPageEnd({ stopFunctie, rawEvents: thisWorkersEvents });
+};
+// #endregion                          MAIN PAGE
+
+// #region [rgba(120, 0, 0, 0.1)]     SINGLE PAGE
+scraper.singlePage = async function ({ page, event }) {
+
+  const { stopFunctie } = await this.singlePageStart();
+
+  const cookiesReq = await page.evaluate(() => {
+    document.querySelector('.overlay.cookie');
+  });
+  if (cookiesReq) {
+    await page.evaluate(() => {
+      document.querySelector('[name*=marketing]').click();
+      document.querySelector('.cookie__process.overlay__button').click();
+    });
+  }
+
+  await this.waitTime(1000);
+
+  const pageInfo = await page
+    .evaluate(
+      // eslint-disable-next-line no-shadow
+      ({ event }) => {
+        const res = {
+          anker: `<a class='page-info' href='${document.location.href}'>${document.title}</a>`,
+          errors: [],
+        };
+
+        try {
+          // TODO: Check whether event is disabled.
+          // if (
+          //   document.querySelector('.event__cta') &&
+          //   document.querySelector('.event__cta').hasAttribute('disabled')
+          // ) {
+          //   res.corrupted += ` ${document.querySelector('.event__cta')?.textContent}`;
+          // }
+        } catch (caughtError) {
+          res.errors.push({
+            error: caughtError,
+            remarks: `check if disabled fail  ${res.anker}`,
+            toDebug: event,
+          });
+        }
+
+        return res;
+      },
+      { event },
+    )
+    .catch((caughtError) => {
+      this.handleError(caughtError, 'pageInfo catch', 'notice', {
+        event,
+        pageInfo,
+      });
+    });
+
+  // TODO: Get door time from .event-data
+  event.startTime = "00:00"
```

Comment:
Als het goed is kan je gewoon de textContent van het relevante element pakken.
Even uit mijn hoofd hoor:
```js
pageInfo.mapToStartTime = await page.evaluate(()=>{
  return document.querySelector('.event-data-left span+ span')?.textContent ?? ''
})
```
en m dan door die functie mapToStartTime ofzo gooien